setup.py

```diff
@@ -117,34 +124,6 @@ def create_version_files(base_dir, version, xla_git_sha, torch_git_sha):
     f.write('}  // namespace torch_xla\n')
 
 
-def generate_xla_lazy_code(base_dir):
-  generate_lazy_cmd = [
-      'python',
-      os.path.join(base_dir, 'scripts', 'gen_lazy_tensor.py')
-  ]
-  child = subprocess.Popen(generate_lazy_cmd)
-  streamdata = child.communicate()[0]
-  if child.returncode != 0:
-    print(
-        'Failed to generate lazy files: {}'.format(generate_lazy_cmd),
-        file=sys.stderr)
-    sys.exit(1)
-
-
-def build_extra_libraries(base_dir, build_mode=None):
-  build_libs_cmd = [os.path.join(base_dir, 'build_torch_xla_libs.sh')]
-  cxx_abi = getattr(torch._C, '_GLIBCXX_USE_CXX11_ABI', None)
```

Comment:
`cxx_abi` is also a param we wanted to control for debugging + and for release. We also had an incident where `torch._C._GLIBCXX_USE_CXX11_ABI` returns an incorrect flag (always false).

Reply:
Cool. Added a `CXX_ABI` flag that can be set from the outside . If empty, it will default to `torch._C`.